shared/AppInsightsCore/src/JavaScriptSDK/InstrumentHooks.ts

```diff
@@ -172,7 +172,7 @@ function _getObjProto(target:any) {
 function _getOwner(target:any, name:string, checkPrototype:boolean): any {
     let owner = null;
     if (target) {
-        if (CoreUtils.hasOwnProperty(target, name)) {
+        if (hasOwnProperty(target, name)) {
```

Comment:
These changes are just a case of stopping direct usage of CoreUtils and use the functions directly